### PR TITLE
fix: highlight in dropdown

### LINKED ISSUE #2401: Bug: blocktype highlight seems to be improper
How do we wish to highlight the selected blockType?

We currently seem to use a span of class active when selected, but this causes the UI to look improper on hovering over other block types as shown below

```
{blockType === 'paragraph' && <span className="active" />}
```

(Observe the background-color size)
<p>

</p>

Lexical version: Latest

## The expected behavior

Please let me know the expected behavior, did we miss adding a tick icon in the active span element or is this the desired behavior itself?

## Note
I would like to work on this!

### PR DESCRIPTION
**Pull request by im-adithya**
Closes #2401 

# Before
<p>

</p>

# After
Screenshot from 2022-06-14 00-24-50

### CODE REVIEW

File: packages/lexical-playground/src/plugins/ToolbarPlugin.tsx [hunk 1]
```diff
@@ -718,50 +718,80 @@ function BlockFormatDropDown({
       buttonIconClassName={'icon block-type ' + blockType}
       buttonLabel={blockTypeToBlockName[blockType]}
       buttonAriaLabel="Formatting options for text style">
-      <DropDownItem className="item" onClick={formatParagraph}>
-        <span className="icon paragraph" />
+      <DropDownItem
+        className={
+          'item ' +
+          (blockType === 'paragraph' ? 'active dropdown-item-active' : '')
```
Comment (fantactuka) [inline packages/lexical-playground/src/plugins/ToolbarPlugin.tsx:724]: Not a big deal, but maybe small helper to take current block type + expected one and output className?

**Review by fantactuka — APPROVED** [review]
LGTM! 🚢

File: packages/lexical-playground/src/plugins/ToolbarPlugin.tsx (hunk 1 above)
Comment (im-adithya) [reply packages/lexical-playground/src/plugins/ToolbarPlugin.tsx:724]: Sure, I was thinking of it as well!

Comment (im-adithya) [reply packages/lexical-playground/src/plugins/ToolbarPlugin.tsx]: Done :D

**Pull request merged**